packages/odyssey-react-mui/src/i18n.ts

```diff
@@ -86,7 +82,6 @@ i18n.use(initReactI18next).init({
   defaultNS,
   ns: [defaultNS],
   fallbackLng: defaultLNG,
-  supportedLngs: supportedLanguages,
```

Comment:
deleting this just means we will try and load any language. As shown in the test, i the language is not supported properly (i.e. doesnt have the proper resources) it defaults to english